gitlint/utils.py

```diff
@@ -64,8 +75,7 @@ def programs_not_in_path(programs):
 def _open_for_write(filename):
     """Opens filename for writing, creating the directories if needed."""
     dirname = os.path.dirname(filename)
-    if not os.path.exists(dirname):
-        os.makedirs(dirname)
+    Path(dirname).mkdir(parents=True, exist_ok=True)
```

Comment:
In the project we use the styleguide from google regarding imports, in which you only import modules. So this would be instead `pathlib.Path`